web-ui/src/streaming/import/random-data/generators.ts

```diff
@@ -0,0 +1,1067 @@
+    generator: () => faker.system.directoryPath()
+  },
+  {
+    title: 'File Path',
+    category: Categories.COMPUTER,
+    generator: () => faker.system.filePath()
+  },
+  {
+    title: 'MIME Type',
+    category: Categories.COMPUTER,
+    generator: () => faker.system.mimeType()
+  },
+  {
+    title: 'Semantic Version',
+    category: Categories.COMPUTER,
+    generator: () => faker.system.semver()
+  },
+  {
+    title: 'Airplane',
+    category: Categories.TRAVEL,
+    generator: () => faker.airline.airplane().name
+  },
+  {
+    title: 'Airplane Code',
+    category: Categories.TRAVEL,
+    generator: () => faker.airline.airplane().iataTypeCode
+  },
+  {
+    title: 'Airlane',
+    category: Categories.TRAVEL,
+    generator: () => faker.airline.airline().name
+  },
+  {
+    title: 'Airlane Code',
+    category: Categories.TRAVEL,
+    generator: () => faker.airline.airline().iataCode
+  },
+  {
+    title: 'Airport',
+    category: Categories.TRAVEL,
+    generator: () => faker.airline.airport().name
+  },
+  {
+    title: 'Airport Code',
+    category: Categories.TRAVEL,
+    generator: () => faker.airline.airport().iataCode
+  },
+  {
+    title: 'Flight number',
+    category: Categories.TRAVEL,
+    generator: () => faker.airline.flightNumber()
+  },
+  {
+    title: 'Record Locator',
+    category: Categories.TRAVEL,
+    generator: () => faker.airline.recordLocator()
+  },
+  {
+    title: 'Airplane Seat',
+    category: Categories.TRAVEL,
+    generator: () => faker.airline.seat()
+  },
+  {
+    title: 'Vehicle',
+    category: Categories.TRAVEL,
+    generator: () => faker.vehicle.vehicle()
+  },
+  {
+    title: 'Vehicle Model',
+    category: Categories.TRAVEL,
+    generator: () => faker.vehicle.model()
+  },
+  {
+    title: 'Vehicle Manufacturer',
+    category: Categories.TRAVEL,
+    generator: () => faker.vehicle.manufacturer()
+  },
+  {
+    title: 'Vehicle Type',
+    category: Categories.TRAVEL,
+    generator: () => faker.vehicle.type()
+  },
+  {
+    title: 'VIN',
+    category: Categories.TRAVEL,
+    generator: () => faker.vehicle.vin()
+  },
+  {
+    title: 'VRM',
+    category: Categories.TRAVEL,
+    generator: () => faker.vehicle.vrm()
+  },
+  {
+    title: 'Bicycle',
+    category: Categories.TRAVEL,
+    generator: () => faker.vehicle.bicycle()
+  },
+  {
+    title: 'Color',
+    category: Categories.PHYSICAL,
+    generator: () => faker.color.human()
+  },
+  {
+    title: 'RGB Color',
+    category: Categories.PHYSICAL,
+    generator: () => faker.color.rgb()
+  },
+  {
+    title: 'Chemical Element Name',
+    category: Categories.PHYSICAL,
+    generator: () => faker.science.chemicalElement().name
+  },
+  {
+    title: 'Chemical Element Symbol',
+    category: Categories.PHYSICAL,
+    generator: () => faker.science.chemicalElement().symbol
+  },
+  {
+    title: 'Unit Name',
+    category: Categories.PHYSICAL,
+    generator: () => faker.science.unit().name
+  },
+  {
```

Comment:
did you really have to type this entire list?
isn't this available in some library?

Reply:
trust me, didn't want to write this either but there wasn't a list